### PR TITLE
Problem: missing a choice of transaction broadcast modes (fixes #513)

### LINKED ISSUE #513: Problem: missing a choice of transaction broadcast modes
currently, the client hardcodes the mode to sync: https://github.com/crypto-com/defi-wallet-core-rs/blob/main/common/src/node/cosmos_sdk.rs#L288
but there are async and commit broadcast modes as well

### PR DESCRIPTION
**Pull request by tomtau**
Solution: extended the UniFFI binding to take a choice of transaction
broadcast modes

### CODE REVIEW

File: common/src/node/cosmos_sdk.rs [hunk 1]
```diff
@@ -287,6 +290,50 @@ pub async fn broadcast_tx_sync(
...
+            .map_err(RestError::RequestError)?
+            .into_result()
+            .map_err(|_e| RestError::MissingResult)?;
+
+        Ok(rpc_result.into())
+    }};
+}
+
+#[cfg(not(target_arch = "wasm32"))]
+fn broadcast_tx_blocking(
```
Comment (damoncro) [inline common/src/node/cosmos_sdk.rs:325]: not familiar with tx broadcast mode, just a little bit confuse/curious why the function name is `blocking` but it also has `async` mode?

Comment (damoncro) [reply common/src/node/cosmos_sdk.rs:325]: I guess if it simply three functions, broadcast_tx_sync_blocking, broadcast_tx_commit_blocking, broadcast_tx_async_blocking...

Comment (damoncro) [reply common/src/node/cosmos_sdk.rs:325]: I know why now, broadcast_tx_blocking and broadcast_tx, both add a new mode parameter.

**Review by damoncro — APPROVED** [review]

File: common/src/node/cosmos_sdk.rs (hunk 1 above)
Comment (tomtau) [reply common/src/node/cosmos_sdk.rs:325]: a bit confusing I guess: it's a mode on the server-side endpoint... so even though it's broadcasted "async", the client execution is still blocking, but the endpoint returns immediately... as you said, it'd be three functions if they all went as separate bindings

**Pull request merged**